utils/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Borrowing this pattern from [react-loosely-lazy](https://github.com/atlassian-labs/react-loosely-lazy/blob/master/babel-plugin/package.json), thanks @MonicaOlejniczak!